src/electron/platform/android/appium-adb-wrapper.ts

```diff
@@ -88,4 +88,20 @@ export class AppiumAdbWrapper implements AdbWrapper {
         this.adb.setDeviceId(deviceId);
         await this.adb.shell(['input', 'keyevent', keyEventCode]);
     };
+
+    public grantOverlayPermission = async (deviceId: string): Promise<void> => {
+        this.adb.setDeviceId(deviceId);
+        await this.adb.shell([
+            'cmd',
+            'appops',
+            'reset',
+            'com.microsoft.accessibilityinsightsforandroidservice',
```

Comment:
The android service configurator has a property for the package name that it uses to pass to other methods on the wrapper. I think we should likely take that same pattern here.